examples/selection/selection/app.py

```diff
@@ -137,6 +155,19 @@ def report_selection(self, widget):
             f"Source: {self.source_selection.value.name} has weight {self.source_selection.value.weight}"
         )
 
+    def change_font(self, widget):
+        self.big_font = not self.big_font
+        if self.big_font:
+            self.styled_selection.style.font_size = 20
+            self.styled_selection.style.font_style = ITALIC
+        else:
+            self.styled_selection.style.font_size = SYSTEM_DEFAULT_FONT_SIZE
+            self.styled_selection.style.font_style = NORMAL
+
+    def print_font_attributes(self, widget):
```

Comment:
I'm not sure this "print" option is adding much - it's not really helpful as a diagnostic, as it's not doing anything but reflecting the values that were set in change_font from the same data source.

Reply:
OK, removed the button and code behind from the Selection example again